ruby_on_rails/bootstrap.md

```diff
@@ -1,15 +1,13 @@
 # Bootstrap
 
-`gem 'bootstrap'`
+You can use the npm package of the latest version of Bootstrap:
 
-Link to the gem: <https://github.com/twbs/bootstrap-rubygem>
+`yarn add bootstrap`
```

Comment:
I'm missing instructions about adding

```javascript
environment.plugins.append('Provide', new webpack.ProvidePlugin({
  $: 'jquery',
  jQuery: 'jquery',
  Popper: ['popper.js', 'default'],
}));
```

to the `environment.js` file as bootstrap requires jquery

Reply:
Maybe also a notice about `bin/webpack-dev-server` would be worth it

Reply:
Furthermore, in `application.js`, `window.$ = $;` should be called, when an included script via a script tag or an inline script requires jQuery.

Reply:
Caesar dixerat webpackeram debet advivere!